net-dns/knot/knot-2.7.6.ebuild

```diff
@@ -0,0 +1,106 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit systemd user
+
+DESCRIPTION="High-performance authoritative-only DNS server"
+HOMEPAGE="https://www.knot-dns.cz/"
+SRC_URI="https://secure.nic.cz/files/knot-dns/${P/_/-}.tar.xz"
+
+LICENSE="GPL-3"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+
+KNOT_MODULES=(
+	"+cookies"
+	"+dnsproxy"
+	"dnstap"
+	"geoip"
+	"+noudp"
+	"+onlinesign"
+	"+queryacl"
+	"+rrl"
+	"+stats"
+	"+synthrecord"
+	"+whoami"
+)
+IUSE="doc caps +fastparser idn libidn2 systemd +utils ${KNOT_MODULES[@]}"
+
+RDEPEND="
+	dev-db/lmdb
+	dev-libs/libedit
+	dev-libs/userspace-rcu:=
+	dev-python/lmdb
+	net-libs/gnutls
+	caps? ( sys-libs/libcap-ng )
+	dnstap? (
+		dev-libs/fstrm
+		dev-libs/protobuf-c
+	)
+	geoip? ( dev-libs/libmaxminddb )
+	idn? (
+		!libidn2? ( net-dns/libidn:* )
+		libidn2? ( net-dns/libidn2 )
+	)
+	systemd? ( sys-apps/systemd )
+"
+BDEPEND="${RDEPEND}
```

Comment:
I'm quite certain those `RDEPEND` should not be in `BDEPEND`. Instead, it seems your ebuild is missing `DEPEND`.

Reply:
Thanks for the review @a17r, I think I misunderstood the changes in EAPI 7. `RDEPEND`s are needed for the `configure` step.

Reply:
Yes, so they should end up in `DEPEND` as well.

Reply:
Right, thanks for the pointer. I just update the commit. I left only `virtual/pkgconfig` and `dev-python/sphinx` which are used on the build-host side.